src/proto/grpc/testing/control.proto

```diff
@@ -220,17 +220,21 @@ message Scenarios {
 // once the scenario has finished.
 message ScenarioResultSummary
 {
-  // Total number of operations per second over all clients.
+  // Total number of oeprations per second over all clients. What is counted as 1 'operation' depends on the benchmark scenarios:
+  // For unary benchmarks, an operation is processing of a single unary RPC. 
+  // For streaming benchmarks, an operation is processing of a single ping pong of request and response. 
   double qps = 1;
-  // QPS per one server core.
+  // QPS per server core.
   double qps_per_server_core = 2;
-  // server load based on system_time (0.85 => 85%)
+  // The overall percentage of server cpu load based on system time across all the server processes.
+  // For example, 85 implies 85% of a cpu core, 125 implies 125% of a cpu core. Since we are accumulating the cpu load across all the server 
+  // processes, the value could > 100 when there are multiple servers. 
```

Comment:
"Since we are accumulating the cpu load across all the server processes, the value could > 100 when there are multiple servers."
Even on a single server the value can be >100% if multiple cores are being used, right?

How about this formulation:
"The total server cpu load based on user time across all server processes, expressed as percentage of a single cpu core ... " + add example values (for both single server and multi-server scenario).

Btw, because we currently have no benchmark scenario with multiple servers, it might make more sense to include the example for multiple clients instead.

Reply:
Right, the cpu load could > 100 when there are multiple cores. Thanks for the phrasing, I have updated the comment and said the example also applies to client cpu load calculation.

Reply:
Probably push commits to a different branch.....sorry for the misleading information. I should have checked this PR earlier, but keep blocking on something else.